ffi/src/android.rs

```diff
@@ -0,0 +1,11 @@
+// TODO just use https://github.com/tomaka/android-rs-glue somehow?
+
+use std::os::raw::c_char;
+use std::os::raw::c_int;
+
+// Logging
+pub const ANDROID_LOG_DEBUG: i32 = 3;
+pub const ANDROID_LOG_INFO: i32 = 4;
+pub const ANDROID_LOG_WARN: i32 = 5;
+pub const ANDROID_LOG_ERROR: i32 = 6;
```

Comment:
I presume there's a good reason for not using an enum with associated values here?

```
pub enum LogLevel {
    Debug = 3,
}
```

… that, I think, will make the values `isize`. Do you really need these to be `i32`?

Reply:
I copied these direct over from storage protos without paying much attention. I'll update.